pyproject.toml

```diff
@@ -54,4 +54,5 @@ pytest = "^7.4.3"
 pytest-dependency = "^0.5.1"
 moto = "^4.2.6"
 pytest-mock = "^3.12.0"
+shellcheck-py = "*"
```

Comment:
Where is that used? Is that something you run manually?

Reply:
It's an optional dependency of actionlint.

Reply:
How does that work? `actionlint` detects automatically that this dependency exists and run it?

Reply:
Exactly.